tests/udfs/test_postprocess.py

```diff
@@ -0,0 +1,148 @@
+import logging
+import os
+import unittest
+from datetime import datetime, timedelta
+from unittest.mock import patch, Mock
+
+import numpy as np
+from fakeredis import FakeServer, FakeStrictRedis
+from freezegun import freeze_time
+from omegaconf import OmegaConf
+from orjson import orjson
+
+from numalogic._constants import TESTS_DIR
+from numalogic.models.autoencoder.variants import SparseVanillaAE
+from numalogic.models.threshold import StdDevThreshold
+from numalogic.registry import RedisRegistry, LocalLRUCache
+from numalogic.tools.exceptions import ModelKeyNotFound
+from numalogic.udfs._config import StreamConf
+from numalogic.udfs.entities import Header, TrainerPayload
+from numalogic.udfs.inference import InferenceUDF
+from numalogic.udfs.postprocess import PostProcessUDF
+from numalogic.udfs.preprocess import PreprocessUDF
+from tests.udfs.utility import input_json_from_file, store_in_redis
+
+logging.basicConfig(level=logging.DEBUG)
+REDIS_CLIENT = FakeStrictRedis(server=FakeServer())
+KEYS = ["service-mesh", "1", "2"]
+DATUM = input_json_from_file(os.path.join(TESTS_DIR, "udfs", "resources", "data", "stream.json"))
+
+
+class TestPostProcessUDF(unittest.TestCase):
```

Comment:
Can you make the tests independent of all other udfs? Look at the inference tests

Reply:
Any reason for it?
The reason I did it this way was to simulate all edge cases like thresh model absent, infer model stale etc etc cause mocking these may not be possible.

Reply:
Because unit tests should be as independent as possible. You can test for all possible edge cases by just changing the input payload.
For the whole pipeline we need to add integration tests

Reply:
Gotcha! Will fix it